docs/software/other/mqtt.md

```diff
@@ -35,6 +36,44 @@ The payload is a raw protobuf. Looking at the MQTT traffic with a program like `
 	ShortFast	!937bed1c
 ```
 
+Packets of the types `TEXT_MESSAGE_APP`, `ENVIRONMENTAL_MEASUREMENT_APP`, `NODEINFO_APP` and `POSITION_APP` are encoded to JSON messages and sent to the `msh/1/json/CHANNELID/DEVICEID` topic. 
```

Comment:
I'd change to wording to something along the lines of:

Messages from the following [port numbers](https://meshtastic.org/docs/developers/firmware/portnum) are serialized to JSON then forwarded to the `msh/1/json/CHANNELID/DEVICEID` topic:
`TEXT_MESSAGE_APP`, `ENVIRONMENTAL_MEASUREMENT_APP`, `NODEINFO_APP` and `POSITION_APP`

Reply:
Fixed in #6dd1de3.